### PR TITLE
Added blockers to all intervention activities.

### PR DESCRIPTION
**Pull request by jrosen081**

### CODE REVIEW

**Review by stefan-philip — APPROVED** [review]

File: frontend/src/pages/Interventions/SecondActivity.tsx [hunk 1]
```diff
@@ -214,26 +215,30 @@ const SecondActivity = ({
...
-              )}
-              {img.correct && index === selectedIndex && (
-                <GreenCircle>
-                  <StyledCheckOutlined />
-                </GreenCircle>
-              )}
-            </ImageContainer>
-          ))}
+          <Blocker afterSeconds={15} message='Click on an image' repeatable={false}>
+            <div>
```
Comment (cindy1u0) [inline frontend/src/pages/Interventions/SecondActivity.tsx]: let's just use <> and </> here since we don't rly need to use styled div

**Review by cindy1u0 — CHANGES_REQUESTED** [review]
LGTM! Just one tiny tiny tiny change

File: frontend/src/pages/Interventions/SecondActivity.tsx (hunk 1 above)
Comment (jrosen081) [reply frontend/src/pages/Interventions/SecondActivity.tsx]: I tried this, but then the blocker does not get the actual click event. I think when we use <> </> then this is like a virtual element that ignores all events, so it does not get sent to the blocker. When there is this div, it then does get the event, which then makes it get the event. This behavior can be tested out here (https://vocab-buddy-1wvn7be9p.vercel.app/interventions/1), this uses not a div. Let me know if you know how I can change this to make it work.

**Review by cindy1u0 — APPROVED** [review]
xd

**Pull request merged**